traitsui/tests/test_labels.py

```diff
@@ -200,6 +220,48 @@ def test_qt_no_labels_on_the_right_bug():
         dialog = NoLabelResizeTestDialog()
         ui = dialog.edit_traits()
 
+@skip_if_not_qt4
+def test_qt_labels_enabled_when():
+    # Behaviour: label should enable/disable along with editor
+
+    with store_exceptions_on_all_threads():
+        dialog = EnableWhenDialog()
+        ui = dialog.edit_traits()
+
+        labelled_editor  = ui.get_editors('labelled_item')[0]
+        unlabelled_editor  = ui.get_editors('unlabelled_item')[0]
+        nose.tools.assert_is_none(unlabelled_editor.label_control)
+
+        nose.tools.assert_true(labelled_editor.label_control.isEnabled())
```

Comment:
I prefer to have these two test functions into a single `unittest.TestCase`

Reply:
Some comment as previous PR - the current test uses nose and I don't want to mix nose and unittest in suites.

Reply:
You can also put them into a plain-old-class named `TestWhatever`. But I wouldn't worry about "mixing" `unittest` and `nose`. `TestCases` are perfectly good `nose` tests.